src/pkg/services/m365/api/drive_test.go

```diff
@@ -4,41 +4,62 @@ import (
 	"testing"
 
 	"github.com/alcionai/clues"
+	"github.com/microsoftgraph/msgraph-sdk-go/models"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 
+	"github.com/alcionai/corso/src/internal/common/ptr"
+	"github.com/alcionai/corso/src/internal/m365/graph"
 	"github.com/alcionai/corso/src/internal/tester"
 	"github.com/alcionai/corso/src/pkg/account"
+	"github.com/alcionai/corso/src/pkg/control"
 	"github.com/alcionai/corso/src/pkg/services/m365/api"
 )
 
-type OneDriveAPISuite struct {
+type DriveAPISuite struct {
 	tester.Suite
-	creds account.M365Config
-	ac    api.Client
+	creds        account.M365Config
+	ac           api.Client
+	driveID      string
+	rootFolderID string
 }
 
-func (suite *OneDriveAPISuite) SetupSuite() {
+func (suite *DriveAPISuite) SetupSuite() {
 	t := suite.T()
+
+	ctx, flush := tester.NewContext(t)
+	defer flush()
+
+	userID := tester.LoadTestM365UserID(t)
```

Comment:
does the load test user currently have data purging setup? Does it need to be enabled to ensure these tests don't take too long? From the test below it seems like this would be fine to use the regular CI user

Reply:
Hah!  🤦 I completely forgot about the naming convention and thought to myself, "why yes, i do want to load the test user id."  Swapping that to the standard user now...